dex/tests/test_rock.py

```diff
@@ -34,8 +34,8 @@ def test_rock(rock_test_env):
     check_rock = CheckRock("rockcraft.yaml")
     rock_image = check_rock.get_name()
     rock_version = check_rock.get_version()
-    LOCAL_ROCK_IMAGE = f"{rock_image}:{rock_version}"
+    LOCAL_rock_IMAGE = f"{rock_image}:{rock_version}"
 
-    # create ROCK filesystem
-    subprocess.run(["docker", "run", LOCAL_ROCK_IMAGE, "exec", "ls", "-la", "/usr/local/bin/docker-entrypoint"], check=True)
-    subprocess.run(["docker", "run", LOCAL_ROCK_IMAGE, "exec", "ls", "-la", "/usr/local/bin/dex"], check=True)
\ No newline at end of file
+    # create rock filesystem
+    subprocess.run(["docker", "run", LOCAL_rock_IMAGE, "exec", "ls", "-la", "/usr/local/bin/docker-entrypoint"], check=True)
```

Comment:
I dont think we should change the env variables. same comment for changes bellow :)